src/mon/MonCommands.h

```diff
@@ -441,6 +441,27 @@ COMMAND("mon remove " \
 COMMAND("mon rm " \
 	"name=name,type=CephString", \
 	"remove monitor named <name>", "mon", "rw", "cli,rest")
+COMMAND("mon features list " \
+        "name=feature_type,type=CephChoices,strings=all|monmap,req=false " \
+        "name=with_value,type=CephChoices,strings=--with-value,req=false", \
+        "list available mon map features to be set/unset", \
+        "mon", "r", "cli")
+COMMAND("mon features set " \
+        "name=feature_name,type=CephString " \
+        "name=feature_type,type=CephChoices,strings=persistent|optional,req=false " \
+        "name=by_value,type=CephChoices,strings=--by-value,req=false " \
```

Comment:
Testing unknown features. But I just realized it will only work on a cluster with one single monitor, because otherwise the monitors will nak each other on probing, and they will shutdown.